src/overlays/actors/ovl_En_Nb/z_en_nb.c

```diff
@@ -745,7 +745,7 @@ void EnNb_InitDemo6KInConfrontation(EnNb* this, PlayState* play) {
 }
 
 void func_80AB2688(EnNb* this, PlayState* play) {
-    this->skelAnime.moveFlags |= 1;
+    this->skelAnime.moveFlags |= ANIM_FLAG_PLAYER_0;
     AnimationContext_SetMoveActor(play, &this->actor, &this->skelAnime, 1.0f);
 }
```

Comment:
(I know nothing about these flags so sorry for the ignorance)
why are these named after player if other actors are using them? its used on their own skelanime, and they pass themselves to "set move actor"

does "set move actor" pertain to player specifically?

Reply:
Only player has code reacting to the `ANIM_FLAG_PLAYER_`-named flags, so it only has a purpose when used in player

I'm not sure what to make of the fact this one is also used outside of player (with no effect afaict)

Reply:
it does make it seem to me like its not intended to be player specific, even if hes the only one who ended up implementing it 
Id prob lean on the safe side and name it more generally, and later on if we feel its right, can name it after him